.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: GitHub CI
+
+on:
+  push:
+    paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.md'
+      - '*.yml'
+  pull_request:
+    paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.md'
+      - '*.yml'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout nxdk
+        uses: actions/checkout@v2
+        with:
+          repository: XboxDev/nxdk
+          submodules: recursive
+      - name: Set xbox_kernel_test_suite dir
+        id: test
+        run: echo "::set-output name=dir::projects/xbox_kernel_test_suite"
+      - name: Checkout xbox_kernel_test_suite
+        uses: actions/checkout@v2
+        with:
+          path: ${{ steps.test.outputs.dir }}
+      - name: Install dependencies
+        run: echo "/usr/lib/llvm-10/bin" >> $GITHUB_PATH
+      - name: Build
+        working-directory: ${{ steps.test.outputs.dir }}
+        run: make -j$(nproc)
+      - name: Set artifact+release parameters
+        id: parameters
+        run: |
+          export file=default.xbe
+          echo "::set-output name=name::$file"
+          echo "::set-output name=path::${{ steps.test.outputs.dir }}/bin/$file"
+          echo "::set-output name=tag::$(date +'%Y%m%d%H%M%S')-$(git log --format=%h -1)"
```

Comment:
For whatever the reason... Format's hash is wrong base on your forked releases page.

Reply:
![image](https://user-images.githubusercontent.com/3938312/113319413-00901a80-92d7-11eb-85dc-fe52e43810ec.png)

Could you somehow explain what happened to your releases page?

Reply:
I don't know why that happens. `git rev-parse --short HEAD` or `${GITHUB_REF:0:7}` could probably be used instead.

Reply:
I'm not sure about `${GITHUB_REF:0:7}` since it may not have a "merged" commit hash? It needs some testing though.

Reply:
I made a typo. GITHUB_REF is the branch, not SHA.
I ended up using `${GITHUB_SHA:0:7}`.